src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -268,6 +270,26 @@ const formConfig = {
           uiSchema: uploadPersonalPtsdDocuments.uiSchema,
           schema: uploadPersonalPtsdDocuments.schema,
         },
+        conclusionCombat: {
+          path: 'conclusion-781',
+          title: 'Disabiity Details',
+          depends: form =>
```

Comment:
Do you think this might make sense to put into `utils` given that the conditions are pretty elaborate?

Reply:
Would also be testable that way ;)

Reply:
That makes sense.